internal/controller/kbsconfig_controller.go

```diff
@@ -416,32 +330,107 @@ func (r *KbsConfigReconciler) newKbsDeployment(ctx context.Context) *appsv1.Depl
 		kbsDeploymentType = confidentialcontainersorgv1alpha1.DeploymentTypeMicroservices
 	}
 
-	// RunAsUser (root) 0
-	runAsUser := int64(0)
 	var volumes []corev1.Volume
+	var kbsVM []corev1.VolumeMount
+	var asVM []corev1.VolumeMount
+	var rvpsVM []corev1.VolumeMount
 
-	// build KBS container
-	volumes, kbsVolumeMounts, err := r.buildKbsVolumeMounts(ctx, volumes)
+	// kbs-config
+	volume, err := r.createKbsConfigMapVolume(ctx, "kbs-config")
```

Comment:
Hmm this is not a refactoring.
It might be better to do this as a separate commit explaining the reason behind the change